### PR TITLE
expand CS7 invariant check

### PR DESCRIPTION
**Pull request by MikeHathaway**
# Description of change
## High level
* Expand CS7 invariant check to better check selection of top slate by votes received.

### CODE REVIEW

File: test/invariants/base/FinalizeInvariants.sol [hunk 1]
```diff
@@ -45,10 +45,12 @@ abstract contract FinalizeInvariants is TestBase {
 
         // check proposal state of the constituents of the top slate
         uint256 totalTokensRequested = 0;
+        int256 topSlateTotalVotesReceived = 0;
         for (uint256 i = 0; i < topSlateProposalIds.length; ++i) {
             uint256 proposalId = topSlateProposalIds[i];
             (, , , uint128 tokensRequested, int128 fundingVotesReceived, ) = grantFund_.getProposalInfo(proposalId);
             totalTokensRequested += tokensRequested;
+            topSlateTotalVotesReceived  += fundingVotesReceived;
```
Comment (EdNoepel) [inline test/invariants/base/FinalizeInvariants.sol]: nit: operator alignment/whitespace

**Review by EdNoepel — APPROVED** [review]
LGTM

**Review by grandizzy — APPROVED** [review]
LGTM

**Review by prat-gpt — APPROVED** [review]
LGTM

**Pull request merged**